docs/Hero.md

```diff
@@ -8,34 +8,30 @@ Hero items are used to call attention to particular values.
 
 ### Hero Usage
 
 ```tsx
 import { Hero } from '@pxblue/react-native-components';
 
-<Hero label={'Charge'} icon={{ family: 'pxblue', name: 'battery' }} value={100} units={'%'} />;
+<Hero label={'Charge'} icon={{ family: 'pxblue', name: 'battery' }} ChannelValueProps={{ value: 100, units: '%' }} />;
 ```
 
 ### Hero API
 
 <div style="overflow: auto">
 
-| Prop Name           | Description                                     | Type                                   | Required | Default                |
-| ------------------- | ----------------------------------------------- | -------------------------------------- | -------- | ---------------------- |
-| label               | The text shown below the `ChannelValue`         | `string`                               | yes      |                        |
-| icon                | A component to render for the primary icon      | [`IconSource`](./Icons.md)             | yes      |                        |
-| iconSize            | The size of the primary icon (10-48)            | `number`                               | no       | 36                     |
-| iconColor           | The color of the primary icon                   | `string`                               | no       | `theme.colors.text`    |
-| iconBackgroundColor | The color behind the primary icon               | `string`                               | no       | `theme.colors.surface` |
-| fontSize            | The text size for the ChannelValue              | `number`                               | no       | 20                     |
-| value               | The value for the ChannelValue                  | `string` \| `number`                   | no       |                        |
-| valueIcon           | A component to render for the ChannelValue icon | [`IconSource`](./Icons.md)             | no       |                        |
-| valueColor          | Color to use for the ChannelValue text          | `string`                               | no       | `text`                 |
-| units               | The units for the ChannelValue                  | `string`                               | no       |                        |
-| onPress             | A function to execute when pressed              | `function`                             | no       |                        |
-| theme               | Theme value overrides                           | `$DeepPartial<ReactNativePaper.Theme>` | no       |                        |
+| Prop Name           | Description                                | Type                                   | Required | Default                |
+| ------------------- | ------------------------------------------ | -------------------------------------- | -------- | ---------------------- |
+| label               | The text shown below the `ChannelValue`    | `string`                               | yes      |                        |
```

Comment:
Changes look good, since you're in here can you alphabetize these props?

Reply:
Yep yep, good call!